.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,57 @@
-
-### Blank / white screen
-If you're getting a blank screen and you report just that, we can do _absolutely_ nothing. By default, your WordPress install suppresses all errors, to prevent information leaks, but we need those errors to be able to help you. If you apply the small piece of code in [this post on WP_DEBUG](https://yoast.com/wordpress-debug/) to your site, you should be able to open the URL that gave you a white screen, append `?debug=debug` to the URL and get the actual error.
-
-That error will help us, without that error, we're completely in the dark about your white page problem...
-
-### Interface errors
-If you're reporting a bug about specific interface elements not working as expected, there's probably an error showing in your browsers JavaScript console. Please open your browsers console and copy the exact error showing there, or make a screenshot. If you don't know how to open your browsers console, here is info for [Chrome](https://developer.chrome.com/devtools/docs/console) and [Firefox](https://developer.mozilla.org/en/docs/Tools/Web_Console). For IE, some Googling will help but it changes with every version.
-
-<a name="security"></a>
-#Security issues
-Please do _not_ report security issues here, but email them to security at yoast dot com so we can deal with them securely and quickly.
-
-<a name="contribute"></a>
-#Contribute to Yoast SEO
-
-Community made patches, localisations, bug reports and contributions are very welcome and help make Yoast SEO remains the #1 SEO plugin for WordPress.
+## I want to create a patch
+Community made patches, localizations, bug reports and contributions are very welcome and help Yoast SEO remain the #1 SEO plugin for WordPress.
 
 When contributing please ensure you follow the guidelines below so that we can keep on top of things.
 
-## Getting Started
-
+#### Getting Started
 If there is no ticket for your issue, submit it first, following the above guidelines.
 
-## Making Changes
+#### Making Changes
 
 * Fork the repository on GitHub (make sure to use the trunk branch, not master).
 * Make the changes to your forked repository.
-  * Ensure you stick to the [WordPress Coding Standards](http://make.wordpress.org/core/handbook/coding-standards/) and have properly documented any new functions, actions and filters following the [documentation standards](http://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/).
+	* Ensure you stick to the WordPress Coding Standards and have properly documented any new functions, actions and filters following the documentation standards.
 * When committing, reference your issue (if present) and include a note about the fix.
 * Push the changes to your fork and submit a pull request to the 'trunk' branch of the Yoast SEO repository.
 
 At this point you're waiting on us to merge your pull request. We'll review all pull requests, and make suggestions and changes if necessary.
```

Comment:
We will review your pull request and merge when everything is in order. We will help you to make sure the code will comply to the standards described above.